docusaurus.config.js

```diff
@@ -141,7 +141,7 @@ const config = {
       {
         name: "node-wot-documentation",
         sourceBaseUrl: "https://raw.githubusercontent.com/eclipse-thingweb/node-wot/refs/heads/master/",
-        outDir: "docs",
+        outDir: "docs/node-wot",
```

Comment:
```suggestion
        outDir: "documentation/node-wot",
```
right? or the name in top bar is controlled by something else?

Reply:
The name in the top bar is controlled in the navbar object, within the config file.

Reply:
I don't understand why it reverted back to docs, but I fixed it again.